scripts/toolbox/config.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
I noticed that you prefer to use modules directly to organize the code. I usually use classes for this, but it's more of a matter of preference. The way you did it works perfectly fine.
I just want to point out that with this approach every module is like a "singleton" class. Modifying imported functionality affects it for the entire scope of the import.

```python
import config

def foo():
    print(config.arangodb.host)

foo()  # localhost
config.arangodb['host'] = 'abcd'
foo()  # abcd
```

Using a separate `config.json` and providing an ArangoDB class, for example, would force the users to work with instances of that class. It also has the benefit of aiding the autocomplete functionality of the IDE, if you provide these keys as class properties.

That being said, I know it may seem like a lot of work now, for just one test. This is not a request for change, but rather a note for the future.